docs/tutorials/google/start.ipynb

```diff
@@ -0,0 +1,559 @@
+        "\n",
+        "`gcloud auth application-default login`\n",
+        "\n",
+        "This will open up a browser window or give you a link to a webpage you can navigate to in order to go through an authentication flow.  Complete this using your @google.com account.  After this command is run, credentials will be stored on your local machine.  If at any point you want to revoke these credentials you can run `gcloud auth application-default revoke`\n",
+        "\n",
+        "We highly recommend that you setup a virtualenv for this installation to isolate your development stack from your overall system installations.  Instructions for virtualenvs are at https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/  Make sure to setup the virtual env for python 3 and not python 2."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "quSdrNseg8LC",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Write and run a short quantum program\n",
+        "\n",
+        "Using your favorite IDE or editor, read and then paste the following hello_qubit program into a file called `hello_qubit.py`.  Make sure to replace the 'your-project-id' string with the project id you created above.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "4RXHE070gleY",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import cirq\n",
+        "\n",
+        "def example():\n",
+        "    \"\"\"Hello qubit example run against a quantum processor.\"\"\"\n",
+        "\n",
+        "    # Define a qubit.\n",
+        "    qubit = cirq.GridQubit(5, 2)\n",
+        "\n",
+        "    # Create a circuit (qubits start in the |0> state).\n",
+        "    circuit = cirq.Circuit(\n",
+        "        cirq.X(qubit)**0.5,                # Square root of NOT.\n",
+        "        cirq.measure(qubit, key='result')  # Measurement.\n",
+        "    )\n",
+        "\n",
+        "    # Create an Engine object.  This uses the project id of your\n",
+        "    # Google cloud project.\n",
+        "    project_id = 'your-project-id'\n",
+        "    engine = cirq.google.Engine(project_id=project_id)\n",
+        "\n",
+        "    print(\"Uploading program and scheduling job on Quantum Engine...\\n\")\n",
+        "\n",
+        "    # Upload the program and submit jobs to run in one call.\n",
+        "    job = engine.run_sweep(\n",
+        "        program=circuit,\n",
+        "        repetitions=1000,\n",
+        "        processor_ids=['rainbow'],\n",
```

Comment:
Should this be `pacific` instead of `rainbow`?

Reply:
Done.